pkg/api/converterfromplugin.go

```diff
@@ -0,0 +1,156 @@
+package api
+
+import (
+	plugin "github.com/openshift/openshift-azure/pkg/api/plugin/api"
+)
+
+func ConvertFromPlugin(in plugin.Config) Config {
```

Comment:
This is more of a merging/overriding function than a conversion and since there is not a conversion back, I would rename this to reflect more appropriately how we use it. Also some godoc would be nice.

Reply:
I agree this is not conversion in generic view. But it does convert one type to other. And it aligns to other files... @jim-minter  WDYT?

Reply:
Ah, misread this, yeah it's a conversion, nvmd